### PR TITLE
added fanart tv

### PR DESCRIPTION
**Pull request by diddy81**
added fanart.tv
fanarttv.py is borrowed from seren with some of my own modifications
if you do decide to merge this feel free to change the api key if you like

### CODE REVIEW

**Comment by drinfernoo** [comment]
How come you closed this?

**Comment by SOOTTV** [comment]
There's some missing artwork and he's trying to fix

**Comment by diddy81** [comment]
Still some things to fix with it I'll reopen it when it's done

**Comment by diddy81** [comment]
fixed a few things i think its all ready to merge give it a test see what you think

File: resources/settings.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
         <setting label="Default TV Show Player" type="text" subsetting="true" id="tvshowsdefault" enable="false" default="None"/>
 		<setting label="Total setup Done" id="total_setup_done" type="bool" default="false" visible="false"/>
 		<setting id="flatten.tvshows" type="enum" label="Flatten TV Show Seasons" values="Never|If only one Season|Always" default="1" />
+		<setting id="enablefanart" type="bool" label="Enable Fanart" default="true" />
```
Comment (drinfernoo) [inline resources/settings.xml:13]: This setting needs to be added back in as well.

**Pull request merged**